wirez-sets/wirez-bpmn/wirez-bpmn-backend/src/test/java/org/wirez/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java

```diff
@@ -308,9 +308,9 @@ public void testUmarshallBoundaryEvents() throws Exception  {
     
     @Test
     public void testUnmarshallSeveralDiagrams() throws Exception  {
-        Diagram<Graph, Settings> diagram1 = unmarshall(BPMN_EVALUATION);
-        assertDiagram( diagram1, 8 );
-        assertEquals( "Evaluation", diagram1.getSettings().getTitle() );
+        Diagram<Graph, Settings> diagram1 = unmarshall(BPMN_LANES);
```

Comment:
This one does not makes sense, you're testing same lanes process twice on the unit test case. In fact, the idea of this test case is testing different marshall/unmarshall operations in a single run, to discard there are state-full component that can cause troubles. So I will let the code as it was :)